migrations/2023-01-09-103911_migrate-watched-to-collection/up.sql

```diff
@@ -0,0 +1,35 @@
+-- Get all unique user id's in watching and create them a 'Watched items collection'
+WITH users_watching AS (
+    SELECT distinct user_id
+    from watched_items
+)
+INSERT
+INTO multiple_collections(created_at, updated_at, deleted_at, user_id, notes, name)
+select now(),
+       now(),
+       null,
+       users_watching.user_id,
+       'Articles you are watching',
+       'Watched items'
+FROM users_watching
+ON CONFLICT DO NOTHING;
+
+-- Add all watched items to that collection. Backup migrated values.
+WITH watching AS (
+    SELECT *
+    from watched_items
+)
+INSERT INTO collection_items (created_at, updated_at, deleted_at, document_id, user_id, notes, custom_name,
+                                multiple_collection_id)
+             SELECT now(),
```

Comment:
any reason this is not `watching.created_at` for both `created_at` and `updated_at`. Otherwise I'd just keep it around.

Reply:
I thought it might be a little odd to migrate these given it's the same data but an entirely differnt context. I can add watching.created_at instead.